operators/aws-neuron-operator/ci.yaml

```diff
@@ -0,0 +1,6 @@
+---
+addReviewers: true
+fbc:
```

Comment:
In order to release your operator to any ocp version, you need to provide a template mapping. Without that the pipeline doesn't know where to publish your operator. See https://redhat-openshift-ecosystem.github.io/operator-pipelines/users/fbc_workflow/#fbc-template-mapping